docs/Parameters.rst

```diff
@@ -1033,11 +1033,10 @@ Objective Parameters
 
    -  used only in ``lambdarank`` application where ``lambdarank_unbiased = true``
 
--  ``lambdarank_eta`` :raw-html:`<a id="lambdarank_eta" title="Permalink to this parameter" href="#lambdarank_eta">&#x1F517;&#xFE0E;</a>`, default = ``0.5``, type = double, constraints: ``lambdarank_eta >= 0.0``
+-  ``lambdarank_bias_p_norm`` :raw-html:`<a id="lambdarank_bias_p_norm" title="Permalink to this parameter" href="#lambdarank_bias_p_norm">&#x1F517;&#xFE0E;</a>`, default = ``0.5``, type = double, constraints: ``lambdarank_bias_p_norm >= 0.0``
```

Comment:
That's a fine name to give the hyperparameter value of p that goes in / specifies the bias norm. 

I had a moment of confusion that this should be a function (p-norm the thing provided, please) and not a hyperparam, so be mindful of the potential name-conflation between the param that says what kind of norming we'll do and the p-norming operation.